src/CanSatKitBMP280.h

```diff
@@ -5,83 +5,132 @@
 
 namespace CanSatKit {
 
+/**
+ * @brief BMP280 pressure sensor base class.
+ */
 class BMP280
 {
 	public:
-		BMP280(); // base type
+		/**
+		 * @brief Construct a new BMP280 object
+		 */
+		BMP280();
 
-		char begin();
-		char begin(int sdaPin, int sclPin);
-			// call pressure.begin() to initialize BMP280 before use
-			// returns 1 if success, 0 if failure (i2C connection problem.)
-				
-		short getOversampling();
-		char  setOversampling(short oss);
-		
-		char startMeasurment();
-			// command BMP280 to start a pressure measurement
-			// oversampling: 0 - 3 for oversampling value
-			// returns (number of ms to wait) for success, 0 for fail
+		/**
+		 * @brief Initialize sensor and calibrate.
+		 * 
+		 * @return `true` if initialization suceeded, `false` otherwise (eg. i2C connection problem)
+		 */
+		bool begin();
 		
-		char calcTemperature(double &T, double &uT);
-			// calculation the true temperature from the given uncalibrated Temperature 
-			
-		char calcPressure(double &P, double uP);
-			//calculation for measuring pressure.
-
-		char getError();
-			// If any library command fails, you can retrieve an extended
-			// error code using this command. Errors are from the wire library: 
-			// 0 = Success
-			// 1 = Data too long to fit in transmit buffer
-			// 2 = Received NACK on transmit of address
-			// 3 = Received NACK on transmit of data
-			// 4 = Other error
-			
-		char readTemperatureAndPressure(double& T, double& P);
+		/**
+		 * @brief Set oversampling ratio.
+		 * Should be one of the following: 0, 1, 2, 3, 4, 16.
+		 * 
+		 * \rst
+		 * +---------------+-------------+
+         * | Oversampling  | Delay (ms)  |
```

Comment:
Consider changing `Delay (ms)` to `Conversion time (ms)`

Reply:
done